addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/status/S3BinaryManagerStatusProbe.java

```diff
@@ -38,20 +39,24 @@
     @Override
     public ProbeStatus run() {
         Map<String, BlobProvider> providers = Framework.getService(BlobManager.class).getBlobProviders();
-        boolean someSuccess = false;
+        boolean found = false;
         for (Map.Entry<String, BlobProvider> en : providers.entrySet()) {
             String id = en.getKey();
             BlobProvider blobProvider = en.getValue();
             BinaryManager bm = blobProvider.getBinaryManager();
-            if (bm == null || !(bm instanceof S3BinaryManager)) {
-                continue;
+            if (bm instanceof S3BinaryManager) {
+                if (!((S3BinaryManager) bm).canAccessBucket()) {
+                    return ProbeStatus.newFailure("S3BinaryManager cannot access the configured bucket: " + id);
+                }
+                found = true;
+            } else if (bm instanceof S3BlobProvider) {
+                if (!((S3BlobProvider) bm).canAccessBucket()) {
+                    return ProbeStatus.newFailure("S3BinaryManager cannot access the configured bucket: " + id);
```

Comment:
I did it on purpose to keep the same message as with the original s3 binary manager, in case we have customer code or tooling parsing the error itself... Do you think it's safe to change it (and backport)?